docs/source/user-guide/vision/vision_properties.rst

```diff
@@ -0,0 +1,188 @@
+===============
+Data Properties
+===============
+
+Some of the Deepchecks' checks (e.g. train-test drift) use "Properties". Properties are simple one-dimension values that
+are extracted either from the images, labels or predictions of the data.
+
+For example, one image property is **brightness**, and one label property is **bounding box area** (for detection
+tasks). Each property produces a single value per image or per individual label in the image.
```

Comment:
I actually dislike both of these paragraphs. I think the structure could be better, as IMO it's weird to begin by "what's using it" instead of "what is it".

My suggestion for these paragraphs:

Image, label or prediction properties are one-dimension values that are extracted either from the data, such as image brightness or label bounding box area. 

These properties are used by some of the Deepchecks' checks (e.g. train-test drift), in order to extract meaningful features over the data (as drift, for example, is difficult to compute directly on images)